### ISSUE TITLE
Glitch with Scout Reports

### ISSUE THREAD

**Issue by kkemp85** [legiongis/hmsf-md#168]
Previously discussed with @mradamcox, we are having issues with new Scout Report submissions. Some of the time, staff and scouts notice that, even when they click save and a new branch show up in the report tree, certain pages of the reports are not being saved or will save as brand new reports completely separate from the original. This causes fragmented Scout Reports to exist in the system. This issue has been happening for a while at least since April.

**Comment by mradamcox** [comment]
@kkemp85 Hopefully the system upgrades a couple of weeks ago fixes this issue. Can you keep an eye out and close this ticket when we haven't encountered the problem for a few more weeks?

**Comment by kkemp85** [comment]
As @mradamcox and I discussed, this issue is still happening, though lately, it is more frequent that pages simply just don't attach, as I experienced myself entering a form the other day in a tutorial.

**Comment by kkemp85** [comment]
I've noticed this is still an issue, still resembling the situation mentioned in the last comment, though I have noticed completely new Scout Reports being created from the glitching pages.

**Comment by mradamcox** [comment]
@kkemp85 Has this occurred recently?

**Comment by kkemp85** [comment]
Yes, though not to me. I have seen it in the recent edit logs as recently as last week. It is hard to tell the difference between user error and this glitch, but I've seen it happen to FPAN staff who know exactly what to do and that this glitch happens.
